### DISCUSSION TITLE
IDEA: Tempmute

### DISCUSSION THREAD

**Discussion by kokosnica** [IgorKowalczyk/majobot#11] [Ideas]
Add command to tempmute/mute user.

**Comment by IgorKowalczyk** [comment]
Good idea, we will start working on moderation module soon. We will definitely add this option! Thank you ^^

**Comment by IgorKowalczyk** [comment]
This feature was added by Discord directly into app: https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ
